packages/common/src/core/schemas/internal/follow/FollowCardSchema.ts

```diff
@@ -0,0 +1,44 @@
+import { IConfigurationSchema } from "../../types";
```

Comment:
This is a very basic schema so we could test the new stencil follow card editor, the props are not accurate and will be modified in [8722](https://zentopia.esri.com/workspaces/sprint-board-60340b9721bc3a4ab6367db5/issues/gh/dc/hub/8722)